test/cli/migrate-unlock.spec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const path = require('path');
+const sqlite3 = require('sqlite3');
+const { expect } = require('chai');
+const { execCommand } = require('cli-testlab');
+const { getRootDir } = require('./cli-test-utils');
+const KNEX = path.normalize(__dirname + '/../../bin/cli.js');
+
+describe('migrate:unlock', () => {
+    before(() => {
+        process.env.KNEX_PATH = '../knex.js';
+    });
+
+    const rootDir = getRootDir();
+    const dbPath = path.resolve(rootDir, 'db');
+
+    it('should restore lock table to one row with is_locked=0', async () => {
+        
+        const db = await new sqlite3.Database(dbPath);
```

Comment:
probably it can also be in-memory db for simplicity as it probably makes no difference in this scenario

Reply:
@kibertoad Can you explain what you mean by "in-memory db" or point me to an example? Is that an internal-only Knex dialect that's only available in test contexts? I haven't an example of that in any other knex tests i've looked at.

Reply:
from one of the knex tests:
```
    beforeEach(() => {
      connection = new sqlite3.Database(':memory:');
    });
```

Reply:
I tried going down the in-memory path but it ended up complicating the test a lot more, as I had to create and tear-down a temporary Knexfile and  in order to have the CLI command write to the correct in-memory DB --- I couldn't get this to work from the CLI:

```execCommand(`node ${KNEX} migrate:unlock --client=sqlite3 --connection=:memory:`);```

Just using the test sqlite DB as the PR currently has in place right now works just fine and seems to be the least-complex option. 

Would you mind if we just retained the current approach?